src/js/components/RangeInput/index.d.ts

```diff
@@ -7,7 +7,7 @@ export interface RangeInputProps {
   min?: number | string;
   max?: number | string;
   name?: string;
-  onChange?: ((event: React.ChangeEvent) => void);
+  onChange?: ((event: React.ChangeEvent<HTMLInputElement>) => void);
```

Comment:
Thank you for this PR! 
Can you please add a few words why you think this fixes the problem?

Reply:
Found a resource here that seems to mirror this solution: https://blaipratdesaba.com/react-typescript-cheatsheet-form-elements-and-onchange-event-types-8c2baf03230c